src/mixins/document-mixin.ts

```diff
@@ -3,8 +3,8 @@ import axios from '@/utils/axios-auth'
 import { Component, Vue } from 'vue-property-decorator'
 import { DocumentUpload } from '@/interfaces'
 import { PdfPageSize } from '@/enums'
-import pdfjsLib from 'pdfjs-dist/build/pdf'
-pdfjsLib.GlobalWorkerOptions.workerSrc = 'public/js/pdf.worker.min.js'
+const pdfjsLib = require('pdfjs-dist/build/pdf')
+pdfjsLib.GlobalWorkerOptions.workerSrc = require('pdfjs-dist/build/pdf.worker.entry')
```

Comment:
I'm sure you played with this, but does the import on line 6 need `require`, or can it follow the usual import syntax?

Reply:
reverted line 6 to back what it was and it seems to work